app/coffee/Features/Project/ProjectDetailsHandler.coffee

```diff
@@ -10,7 +10,7 @@ module.exports =
 
 	getDetails: (project_id, callback)->
 		ProjectGetter.getProject project_id, {name:true, description:true, compiler:true, features:true, owner_ref:true}, (err, project)->
-			if err?
+			if err? or !project?
```

Comment:
If you just want a quick fix, this is fine. But the 'correct' way to do it would be include './Errors' and return a NotFoundError, which will send back a 404 from the HTTP request, and not show up in Sentry etc.